src/main/flight/pid.h

```diff
@@ -76,13 +83,16 @@ typedef struct pidProfile_s {
 
 PG_DECLARE_PROFILE(pidProfile_t, pidProfile);
 
+static inline const pid8_t * pidBank() { return STATE(FIXED_WING) ? pidProfile()->bank_fw : pidProfile()->bank_mc; }
```

Comment:
@martinbudden I think `pidBank(PID_ROLL)` is somewhat unintuitive. I would prefer `pidBank` to return a structure. I believe some other parameters might be part of `PID bank` as well, i.e. `dterm_setpoint_weight` or `yaw_p_limit`.